test/reduce.jl

```diff
@@ -361,15 +349,6 @@ end
 
 ## cumsum, cummin, cummax
 
-z = rand(10^6)
-let es = sum_kbn(z), es2 = sum_kbn(z[1:10^5])
-    @test (es - sum(z)) < es * 1e-13
-    cs = cumsum(z)
-    @test (es - cs[end]) < es * 1e-13
-    @test (es2 - cs[10^5]) < es2 * 1e-13
-end
```

Comment:
I wonder whether these tests' purpose is testing `[cum]sum` accuracy rather than testing `sub_kbn` per se? And if so, perhaps retaining these tests with some other mechanism to calculate an accurate sum would be worthwhile?

Reply:
Like using `BigFloat`?